styles/utility.css

```diff
@@ -414,3 +414,241 @@ textarea {
+    margin-right: 100px;
+  }
+
+  .desktop_nav_wrapper {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+  }
+
+  .desktop_nav_li {
+    margin-right: 42px;
+  }
+
+  .desktop_nav_li a {
+    text-decoration: none;
+    color: #172b4d;
+    font-weight: 700;
+  }
+
+  .desktop_nav_li a:hover {
+    color: #172b4d;
+    font-size: large;
+  }
+
+  .hamburger {
+    display: none;
+  }
+
+  .logo {
+    display: none;
+  }
+
+  section.hero {
+    background-image: url('../images/Header-llustration-desktop.png');
+    background-repeat: no-repeat;
+    width: 100%;
+    height: 1250px;
+    position: relative;
+  }
+
+  .headline-desc {
+    display: flex;
+    flex-flow: column;
+    align-items: center;
+    width: 100%;
+    padding-top: 170px;
+  }
+
+  .headline-desc h1 {
+    width: 700px;
+    font-size: 48px;
+    line-height: 60px;
+  }
+
+  .headline-desc p {
+    width: 60%;
+    line-height: 28px;
+    font-size: 20px;
+    align-items: center;
+    justify-content: center;
+    letter-spacing: 0.89px;
+    margin-top: -30px;
+  }
+
+  .allprojects-container {
+    display: flex;
+    flex-direction: row;
+    flex-wrap: wrap;
+    justify-content: space-evenly;
+    margin: 0 4.5vw;
+  }
+
+  .project-container-card {
+    width: 28vw;
+    height: 50vh;
+  }
+
+  .programming-skills-container {
+    display: flex;
+    justify-content: space-evenly;
+    margin: 0 3.5vw;
+  }
+
+  .programming-skills {
+    width: 28vw;
+  }
+
+  .about-me-container {
+    position: relative;
+    top: -70px;
+    display: flex;
+    flex-flow: column;
+    align-items: center;
+    width: 100%;
+    background-image:
+      url("../images/illustration-about-me-2-desktop.png"),
+      url("../images/illustration\ about\ me\ desktop.png");
+    background-position: left;
+    background-repeat: no-repeat;
+  }
+
+  .about-me-icon {
+    display: none;
+  }
+
+  .about-me-container p {
+    width: 920px;
+    height: 96px;
+    line-height: 32px;
+    font-size: 20px;
+    align-items: center;
+    justify-content: center;
+    letter-spacing: 1.8px;
+    color: #172b4d;
+  }
+
+  .wrapper {
+    margin-bottom: 150px;
+  }
+
+  .form-wrapper {
+    margin-bottom: 150px;
+    background-image: url("../images/illustration-1-contact-form-desktop.png");
+    background-repeat: no-repeat;
+    background-position: left 24vh;
+  }
+
+  .btn.btn-projects {
+    font-size: 20px;
+    letter-spacing: 0.03em;
+    text-align: center;
+    border-radius: 4px;
+    align-self: center;
+  }
+
+  .footer-img {
+    content: url("../images/contact\ form-illustratoin-3-desktop.png");
+  }
+
+  .contact-me {
+    margin-top: 152px;
+    display: flex;
+    flex-direction: row;
+    position: relative;
+    justify-content: space-around;
+    background-image: url("../images/illustration-2-contact-form-desktop.png");
+    background-repeat: no-repeat;
+    background-position-y: bottom;
+    background-position-x: 20vw;
+  }
```

Comment:
- [x] Great job so far! But unfortunately, you need to stick to the design as much as possible for better UI (user interface) and this is a good professional practice that every frontend software developer should maintain. Please, make sure to give your background image some paddings on the left side so they can look closely similar to the [original design](https://www.figma.com/file/t3EJUCAEViw3QasuJLPLVT/Microverse-Student-Potfolio-Templates-Main?node-id=1%3A1471) 😸 .
- [ ] As you can see in screenshots below you have some overflow following `x-direction` please try to work on this issue by making it hidden

## **Your project**
![real](https://user-images.githubusercontent.com/80895497/150617256-c011a221-1b20-4aff-b2b1-0697f9ebf25b.PNG)

## **Design**
![design](https://user-images.githubusercontent.com/80895497/150617254-41989132-0651-415c-882c-e0266e72e15e.PNG)

Reply:
Hi @Hamzaoutdoors 

I have worked on the overflow and padding issues just as it is in the design.
Thanks for your review